### PR TITLE
Upgrading to 5.5.1 to use latest releases on central-ledger and centr…

### PR DESCRIPTION
**Pull request by HenkKodde**
As per story #608 - Add config to default.json to manage the connection pool for knex and MySQL.

### CODE REVIEW

**Comment by elnyry** [comment]
@HenkKodde this sounds contrary to Rajiv's comment above, but since the Sprint is 6.1 now, can you please either update the version numbers accordingly or remove the version updates altogether?

File: centralledger/chart-handler-admin-transfer/configs/knexfile.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-admin-transfer/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-admin-transfer/configs/knexfile.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-admin-transfer/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-timeout/configs/knexfile.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-timeout/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-timeout/configs/knexfile.js [hunk 4]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-timeout/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-transfer-fulfil/configs/knexfile.js [hunk 5]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-fulfil/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-transfer-fulfil/configs/knexfile.js [hunk 6]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-fulfil/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-transfer-get/configs/knexfile.js [hunk 7]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-get/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-transfer-get/configs/knexfile.js [hunk 8]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-get/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-transfer-position/configs/knexfile.js [hunk 9]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-position/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-transfer-position/configs/knexfile.js [hunk 10]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-position/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-transfer-prepare/configs/knexfile.js [hunk 11]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-prepare/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-handler-transfer-prepare/configs/knexfile.js [hunk 12]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-prepare/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-service/configs/knexfile.js [hunk 13]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```
Comment (mdebarros) [inline centralledger/chart-service/configs/knexfile.js]: This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

File: centralledger/chart-service/configs/knexfile.js [hunk 14]
```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
...
+    max: Config.DB_CONNECTION.DB_CONNECTION_POOL_MAX,
```
Comment (mdebarros) [inline centralledger/chart-service/configs/knexfile.js]: This should be: `max: Config.DB_CONNECTION_POOL_MAX,`

File: centralledger/chart-handler-admin-transfer/configs/knexfile.js (hunk 1 above)
Comment (HenkKodde) [reply centralledger/chart-handler-admin-transfer/configs/knexfile.js]: If we apply the same updated as requested for central_ledger, should it not be min: Config.POOL_MIN?

Comment (HenkKodde) [reply centralledger/chart-handler-admin-transfer/configs/knexfile.js]: Thank you - can see the error and will updated.

File: centralledger/chart-handler-admin-transfer/configs/knexfile.js (hunk 2 above)
Comment (HenkKodde) [reply centralledger/chart-handler-admin-transfer/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-timeout/configs/knexfile.js (hunk 3 above)
Comment (HenkKodde) [reply centralledger/chart-handler-timeout/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-timeout/configs/knexfile.js (hunk 4 above)
Comment (HenkKodde) [reply centralledger/chart-handler-timeout/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-fulfil/configs/knexfile.js (hunk 5 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-fulfil/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-fulfil/configs/knexfile.js (hunk 6 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-fulfil/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-get/configs/knexfile.js (hunk 7 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-get/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-get/configs/knexfile.js (hunk 8 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-get/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-position/configs/knexfile.js (hunk 9 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-position/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-position/configs/knexfile.js (hunk 10 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-position/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-prepare/configs/knexfile.js (hunk 11 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-prepare/configs/knexfile.js]: Updated as requested.

Comment (HenkKodde) [reply centralledger/chart-handler-transfer-prepare/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-handler-transfer-prepare/configs/knexfile.js (hunk 12 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-prepare/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-service/configs/knexfile.js (hunk 13 above)
Comment (HenkKodde) [reply centralledger/chart-service/configs/knexfile.js]: Updated as requested.

File: centralledger/chart-service/configs/knexfile.js (hunk 14 above)
Comment (HenkKodde) [reply centralledger/chart-service/configs/knexfile.js]: Updated as requested.

File: central/Chart.yaml [hunk 15]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central Helm chart for Kubernetes
 name: central
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline central/Chart.yaml]: this should be 6.1.0

File: central/requirements.yaml [hunk 16]
```diff
@@ -1,7 +1,7 @@
 # requirements.yaml
 dependencies:
 - name: centralledger
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline central/requirements.yaml]: this should be 6.1.0

File: centralledger/Chart.yaml [hunk 17]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Helm chart for Kubernetes
 name: centralledger
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-admin-transfer/Chart.yaml [hunk 18]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Transfer Prepare Handler Helm chart for Kubernetes
 name: centralledger-handler-admin-transfer
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-handler-admin-transfer/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-timeout/Chart.yaml [hunk 19]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Timeout Handler Helm chart for Kubernetes
 name: centralledger-handler-timeout
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-handler-timeout/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-transfer-fulfil/Chart.yaml [hunk 20]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Transfer Fulfil Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-fulfil
-version: 6.1.0
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-fulfil/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-transfer-get/Chart.yaml [hunk 21]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Get Transfer Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-get
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-get/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-transfer-position/Chart.yaml [hunk 22]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Transfer Position Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-position
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-position/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-handler-transfer-prepare/Chart.yaml [hunk 23]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Transfer Prepare Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-prepare
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-prepare/Chart.yaml]: this should be 6.1.0

File: centralledger/chart-service/Chart.yaml [hunk 24]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Central-Ledger Service Helm chart for Kubernetes
 name: centralledger-service
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline centralledger/chart-service/Chart.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 25]
```diff
@@ -1,31 +1,31 @@
 # requirements.yaml
 dependencies:
 - name: centralledger-service
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 26]
```diff
@@ -1,31 +1,31 @@
...
   repository: "file://./chart-service"
   condition: centralledger-service.enabled
 - name: centralledger-handler-transfer-prepare
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 27]
```diff
@@ -1,31 +1,31 @@
...
   repository: "file://./chart-handler-transfer-prepare"
   condition: centralledger-handler-transfer-prepare.enabled
 - name: centralledger-handler-transfer-position
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 28]
```diff
@@ -1,31 +1,31 @@
...
   repository: "file://./chart-handler-transfer-position"
   condition: centralledger-handler-transfer-position.enabled
 - name: centralledger-handler-transfer-get
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 29]
```diff
@@ -1,31 +1,31 @@
...
   repository: "file://./chart-handler-transfer-get"
   condition: centralledger-handler-transfer-get.enabled
 - name: centralledger-handler-transfer-fulfil
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/requirements.yaml [hunk 30]
```diff
@@ -1,31 +1,31 @@
...
   repository: "file://./chart-handler-transfer-fulfil"
   condition: centralledger-handler-transfer-fulfil.enabled
 - name: centralledger-handler-timeout
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline centralledger/requirements.yaml]: this should be 6.1.0

File: centralledger/values.yaml [hunk 31]
```diff
@@ -30,7 +30,7 @@ centralledger-service:
     api:
       image:
         repository: mojaloop/central-ledger
-        tag: v6.1.0
+        tag: v6.1.1
```
Comment (mdebarros) [inline centralledger/values.yaml]: this should be 6.1.0

File: mojaloop/Chart.yaml [hunk 32]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Mojaloop Helm chart for Kubernetes
 name: mojaloop
-version: 6.1.0
-appVersion: "6.1.0"
+version: 6.1.1
```
Comment (mdebarros) [inline mojaloop/Chart.yaml]: this should be 6.1.0

File: mojaloop/requirements.yaml [hunk 33]
```diff
@@ -1,7 +1,7 @@
 # requirements.yaml
 dependencies:
 - name: central
-  version: 6.1.0
+  version: 6.1.1
```
Comment (mdebarros) [inline mojaloop/requirements.yaml]: this should be 6.1.0

File: central/Chart.yaml (hunk 15 above)
Comment (HenkKodde) [reply central/Chart.yaml]: Updated - version: 6.1.0

File: central/requirements.yaml (hunk 16 above)
Comment (HenkKodde) [reply central/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/Chart.yaml (hunk 17 above)
Comment (HenkKodde) [reply centralledger/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-admin-transfer/Chart.yaml (hunk 18 above)
Comment (HenkKodde) [reply centralledger/chart-handler-admin-transfer/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-timeout/Chart.yaml (hunk 19 above)
Comment (HenkKodde) [reply centralledger/chart-handler-timeout/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-transfer-get/Chart.yaml (hunk 21 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-get/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-transfer-fulfil/Chart.yaml (hunk 20 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-fulfil/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-transfer-get/Chart.yaml (hunk 21 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-get/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-transfer-position/Chart.yaml (hunk 22 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-position/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-handler-transfer-prepare/Chart.yaml (hunk 23 above)
Comment (HenkKodde) [reply centralledger/chart-handler-transfer-prepare/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/chart-service/Chart.yaml (hunk 24 above)
Comment (HenkKodde) [reply centralledger/chart-service/Chart.yaml]: Updated - version: 6.1.0

File: centralledger/requirements.yaml (hunk 25 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0 (7 places)

File: centralledger/requirements.yaml (hunk 26 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/requirements.yaml (hunk 27 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/requirements.yaml (hunk 28 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/requirements.yaml (hunk 29 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/requirements.yaml (hunk 30 above)
Comment (HenkKodde) [reply centralledger/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/values.yaml (hunk 31 above)
Comment (HenkKodde) [reply centralledger/values.yaml]: Updated - version: 6.1.0

File: mojaloop/Chart.yaml (hunk 32 above)
Comment (HenkKodde) [reply mojaloop/Chart.yaml]: Updated - version: 6.1.0

File: mojaloop/requirements.yaml (hunk 33 above)
Comment (HenkKodde) [reply mojaloop/requirements.yaml]: Updated - version: 6.1.0

File: centralledger/values.yaml (hunk 31 above)
Comment (HenkKodde) [reply centralledger/values.yaml]: Not sure about this one. all other tag: are currently 6.1.1 related to central-ledger.

**Pull request merged**